keyboards/atreyu/rev1/rev1.c

```diff
@@ -13,35 +13,3 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-
```

Comment:
Why is this being removed? 

If the board supports encoders, this should be left in place.